src/sqlfluff/rules/L030.py

```diff
@@ -48,13 +51,18 @@ def _get_fix(self, segment, fixed_raw):
         We need to do this because function_name nodes have a child
         function_name_identifier that holds the actual name.
         """
-        child_segment = [
-            child
-            for child in segment.segments
-            if child.get_type() == "function_name_identifier"
-        ][0]
-        return LintFix(
-            "edit",
-            child_segment,
-            child_segment.edit(fixed_raw.strip()),
-        )
+
```

Comment:
Black is complaining about this blank line (though can't see it as CircleCI not running)

```suggestion
```

Reply:
Done